packages/fiori/src/IllustratedMessage.ts

```diff
@@ -153,6 +154,20 @@ class IllustratedMessage extends UI5Element {
 	@property({ defaultValue: "" })
 	accessibleNameRef!: string;
 
+	/**
+	* Defines the semantic level of the title.
+	*
+	* <b>Note:</b> Used for accessibility purposes only.
+	*
+	* @type {sap.ui.webc.fiori.types.TitleLevel}
+	* @defaultvalue "TitleLevel.H2"
```

Comment:
H2 instead of TitleLevel.H2

Reply:
argTypes are automatically generated from custom-elements.json which depends on correct JSDoc. Here we have mistake because the type is set to `sap.ui.webc.fiori.types.TitleLevel` but TitleLevel types is based in the main package (it has to be `sap.ui.webc.main.types.TitleLevel`